### PR TITLE
Lista4 komponenty

### PR DESCRIPTION
**Pull request by wiktorkowalski**

### CODE REVIEW

File: lista4/Komponenty/zadanie3/Service1.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+        protected override void OnStart(string[] args)
+        {
+            //var temp = MessageBox.Show("AWS","Amazing Windows Service ON");
+        }
+
+        protected override void OnStop()
+        {
+            //var temp = MessageBox.Show("AWS", "Amazing Windows Service OFF");
+        }
```
Comment (mikolajkalwa) [inline lista4/Komponenty/zadanie3/Service1.cs:29]: commented code
both methods do nothing

File: lista4/Komponenty/ClockComponent/Class1.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ClockComponent
+{
+    public class Class1
+    {
+    }
```
Comment (mikolajkalwa) [inline lista4/Komponenty/ClockComponent/Class1.cs]: What is it?

File: lista4/Komponenty/TrafficComponent/Class1.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrafficComponent
+{
+    public class Class1
+    {
+
+    }
```
Comment (mikolajkalwa) [inline lista4/Komponenty/TrafficComponent/Class1.cs]: What is it?

File: lista4/Komponenty/zadanie1/Form1.cs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        public Form1()
+        {
+            InitializeComponent();
+
+        }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            
+        }
```
Comment (mikolajkalwa) [inline lista4/Komponenty/zadanie1/Form1.cs]: can be removed (?)

File: lista4/Komponenty/zadanie3/Service1.cs (hunk 1 above)
Comment (wiktorkowalski) [reply lista4/Komponenty/zadanie3/Service1.cs:29]: Doesn't work and makes the service fail to start, but i wanna keep it to show where actual code can go.

**Review by mikolajkalwa — APPROVED** [review]

**Pull request merged**